pymeasure/instruments/srs/sr830.py

```diff
@@ -414,9 +414,9 @@ def output_conversion(self, channel):
         """ Returns a function that can be used to determine
         the signal from the channel output (X, Y, or R)
         """
-        offset, expand = self.get_scaling(channel)
+        offset, _ = self.get_scaling(channel)
         sensitivity = self.sensitivity
-        return lambda x: (x / (10. * expand) + offset) * sensitivity
+        return lambda x: x  + offset / 100 * sensitivity
```

Comment:
thanks for pushing for writing the tests. it forced me to test this with hardware and I found that even my changed version was not correct. It turns out that the expand value should not be used at all in the conversion function. The latest version now behaves as expected with my hardware.

Reply:
Thanks for writing the tests.
I'm glad to hear that the tests improved code quality yet another time.

I don't know how many bugs I discovered by having tests.